### PR TITLE
New Template: Cisco XR - show arp

### PR DESCRIPTION
**Pull request by migueloangelo**
##### ISSUE TYPE

 - New Template Pull Request [X]
 - Bugfix Pull Request
 - Additional Testing
 - Docs Pull Request

##### COMPONENT

Name: cisco_xr_show_arp
OS: cisco_xr
Command: show arp

##### SUMMARY

Template for the ```show arp``` command in Cisco XR. Tested with Cisco IOS XRv version 6.0.1. Also support the ```show arp vrf [VRFNAME]``` command.

```

```

### CODE REVIEW

File: templates/cisco_xr_show_arp.textfsm [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Value Required AGE (\S+)
+Value Required MAC (\S+)
+Value Required STATE (\S+)
+Value Required TYPE (\S+)
+Value Required INTERFACE (\S+)
+Value Filldown CPU (\d+/\d+/CPU\d+$)
+
+Start
+  ^\S+\s+\S+\s+\d+\s+\d+:\d+:\d+\.\d+\s+\S+\s*$$
+  ^Address\s+Age\s+Hardware Addr\s+State\s+Type\s+Interface$$
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  ^Address\s+Age\s+Hardware\s+Addr\s+State\s+Type\s+Interface$$
```

File: templates/cisco_xr_show_arp.textfsm [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  ^-+$$
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  ^-+
```

File: templates/cisco_xr_show_arp.textfsm [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  ^${CPU}$$
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  ^${CPU}
```

File: templates/cisco_xr_show_arp.textfsm [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  ^\s$$
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  ^\s*$$
```

File: templates/cisco_xr_show_arp.textfsm [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  ^${ADDRESS}\s+${AGE}\s+${MAC}\s+${STATE}\s+${TYPE}\s+${INTERFACE}$$ -> Record
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  ^${ADDRESS}\s+${AGE}\s+${MAC}\s+${STATE}\s+${TYPE}\s+${INTERFACE}\s*$$ -> Record
```

File: templates/cisco_xr_show_arp.textfsm [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Value Required ADDRESS (\d+\.\d+\.\d+\.\d+)
+Value Required AGE (\S+)
+Value Required MAC (\S+)
+Value Required STATE (\S+)
+Value Required TYPE (\S+)
+Value Required INTERFACE (\S+)
+Value Filldown CPU (\d+/\d+/CPU\d+$)
+
+Start
+  ^\S+\s+\S+\s+\d+\s+\d+:\d+:\d+\.\d+\s+\S+\s*$$
```
Comment (jmcgill298) [inline templates/cisco_xr_show_arp.textfsm]:
```suggestion
  # Match the timestamp at beginning of command output
  ^\S+\s+\S+\s+\d+\s+\d+:\d+:\d+\.\d+\s+\S+\s*$$
```

**Comment by migueloangelo** [comment]
Done, thanks!

**Pull request merged**